### PR TITLE
made volunteer list dynamic

### PR DESCRIPTION
**Pull request by kybrdbnd**
This PR makes volunteer list dynamic

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,58 +14,41 @@
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/3.0.3/normalize.min.css">
     <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
     <link href='https://pydelhi.org/assests/css/main.css' rel='stylesheet'>
+    <script src="https://code.jquery.com/jquery-3.5.1.js"></script>
```
Comment (Animesh-Ghosh) [inline index.html:17]: Why did we need `jQuery`? Can't we make do with vanilla JavaScript?

Comment (kybrdbnd) [reply index.html:17]: working with dynamic content requires jQuery and hence its usage, "$" is a part of jQuery

Comment (kybrdbnd) [reply index.html:17]: > working with dynamic content requires jQuery and hence its usage, "$" is a part of jQuery

working with dynamic content requires jQuery and hence its usage, "$" is a part of jQuery

Comment (kybrdbnd) [reply index.html:17]: > Why did we need `jQuery`? Can't we make do with vanilla JavaScript?

working with dynamic content requires jQuery and hence its usage, "$" is a part of jQuery

Comment (Animesh-Ghosh) [reply index.html:17]: We can use the browser's [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) API instead. Decreases dependencies.

**Comment by satyaakam** [comment]
Nice can we extend it further add some CSS to the final rendered page , also in the JSON add a picture and twitter account in the JSON and update the PR

File: index.html (hunk 1 above)
Comment (kybrdbnd) [reply index.html:17]: as discussed earlier, using of JQuery is fine

**Comment by kybrdbnd** [comment]
> Nice can we extend it further add some CSS to the final rendered page , also in the JSON add a picture and twitter account in the JSON and update the PR

happy to raise another PR for this request

**Review by Animesh-Ghosh — APPROVED** [review]

**Review by satyaakam — APPROVED** [review]

**Pull request merged**